programas.html

```diff
@@ -1,16 +1,45 @@
 ---
 layout: default
 ---
+<head>
```

Comment:
Add custom css into https://github.com/fundacionmamasena/fundacionmamasena.github.io/blob/master/css/custom.css, so we don't have to insert two <head> tags in the page